### DISCUSSION TITLE
UnityWebBrowser packages not showing up

### DISCUSSION THREAD

**Discussion by LEVAAP** [Voltstro-Studios/UnityWebBrowser#377] [Q&A] [answered]
Set up everything (right scopes), but UnityWebBrowser packages don't show up
<img width="515" alt="Schermafbeelding 2024-12-27 154031" src="https://github.com/user-attachments/assets/bc1f2d00-ba0b-43e8-afbb-05b766470c3b" />
<img width="562" alt="Schermafbeelding 2024-12-27 154003" src="https://github.com/user-attachments/assets/6041a278-1161-4cf0-bed2-cfa493d761a4" />

**Comment by Voltstro** [comment, marked as answer]
Miss-spelled `voltstro`. You have typed it as `dev.volstro`, it should be `dev.voltstro`.

**Reply by LEVAAP** [reply]
Haha, I feel so stupid. Thanks!

**Reply by LEVAAP** [reply]
Hi, it's me again. Unity Web Browser shows up now, but the engine and browsers won't :(
![afbeelding_2024-12-27_180340472](https://github.com/user-attachments/assets/a05e376e-71e3-468d-b4e7-e7c5e3fa7f5a)

**Reply by Voltstro** [reply]
Not too sure why, but just try to manually add the packages. You can do that in the package manager by pressing the + symbol, "Add package by name".

![image](https://github.com/user-attachments/assets/9f75263b-cf0e-42ef-9698-ed80a7f5f141)

Two packages you probably want are `dev.voltstro.unitywebbrowser.engine.cef` and `dev.voltstro.unitywebbrowser.engine.cef.win.x64`.

**Reply by LEVAAP** [reply]
Found them! Thanks! They were hiding under 'pre-release'
